app/ninja/invoice.py

```diff
@@ -55,7 +55,7 @@ def delete_item(self, invoice_items):
         self.invoice_items = [
             data
             for data in self.invoice_items
-            if data["invoice_items"] != invoice_items
+            if data["notes"] != invoice_items['notes']
```

Comment:
Better use `data.get('notes')` and `invoice_items.get('notes')` to avoid crashes